app/src/test/java/com/epfl/drawyourpath/friendsList/FriendsViewModelTest.kt

```diff
@@ -0,0 +1,50 @@
+package com.epfl.drawyourpath.friendsList
+
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import androidx.lifecycle.Observer
+import com.epfl.drawyourpath.R
+import com.epfl.drawyourpath.mainpage.fragments.helperClasses.Friend
+import com.epfl.drawyourpath.mainpage.fragments.helperClasses.FriendsViewModel
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mockito.Mockito.*
+
+class FriendsViewModelTest {
+

```

Comment:
Optional : Maybe you can describe with comments a little more your test, to understand what the test is doing.